pf9/cluster/commands.py

```diff
@@ -89,6 +93,8 @@ def attach_cluster(cluster_name, master_ips, worker_ips, ctx):
                     ctx.params['cluster_name']), fg="red")
         sys.exit(1)
 
+    SegmentSessionWrapper(ctx).send_track("Validate Cluster Existence")
```

Comment:
Shouldn't these be before the actual operation to indicate that is what is happening? Or is it to be interpreted as what step has finished.

Reply:
Yea I guess it can interpreted as what step has finished, also when we add tracking of Error cases we would know if the CLI execution has failed for some reason.